### PR TITLE
upload again

### PR DESCRIPTION
**Pull request by rohanasif**

### CODE REVIEW

File: covidanalyzer.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+parser = argparse.ArgumentParser()
+# Add an argument
+parser.add_argument('-a', type=str)
+parser.add_argument('-b', type=str)
+parser.add_argument('-c', type=str)
+# Parse the argument
+args = parser.parse_args()
+
+
+if args.a is not None:
```
Comment (uzair-ahmed-global) [inline covidanalyzer.py]: Look up truthy and falsy values in python.

File: results_calculator.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import argparse
+from FileReader import FileReader
+
+# Part (a) calculations
+def recovered_over_total(args):
```
Comment (uzair-ahmed-global) [inline results_calculator.py]: Let's implement this whole module as a class.

File: results_calculator.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    #print(countries_index)
+    for index in countries_index:
+        total_cases += total_cases_in_each_country[index]
+        total_deaths += total_deaths_in_each_country[index]
+
+    print(str((total_deaths/total_cases)*100) + "% death average found in " +
+          (str(len(countries_following_required_measure))) + " countries")
+
+# Part (c) calculations
+# def top_5_efficiencies(args):
```
Comment (uzair-ahmed-global) [inline results_calculator.py]: Please remove commented code. And add a newline at end of file.

**Review by uzair-ahmed-global — CHANGES_REQUESTED** [review]

File: covidanalyzer.py (hunk 1 above)
Comment (rohanasif) [reply covidanalyzer.py]: ok

File: results_calculator.py (hunk 2 above)
Comment (rohanasif) [reply results_calculator.py]: what do you mean?...I have used FileReader as a module of FileReader class here...

File: results_calculator.py (hunk 3 above)
Comment (rohanasif) [reply results_calculator.py]: ok

File: results_calculator.py (hunk 2 above)
Comment (uzair-ahmed-global) [reply results_calculator.py]: Just like that, make a module out of this results_calculator file as well. For now it's just a file with functions in it. Turn it into a file with a class in it that has these functions.

File: covidanalyzer.py (hunk 1 above)
Comment (rohanasif) [reply covidanalyzer.py]: corrected the code with the truthy and falsy syntax

File: results_calculator.py (hunk 2 above)
Comment (rohanasif) [reply results_calculator.py]: well now the functions recovered_over_total(args) and measures_death_rate(args) are not recognized after importing the class made in results_calculator

**Comment by rohanasif** [comment]
reopened

**Pull request merged**